projects/plugins/jetpack/modules/subscriptions/jetpack-user-content-link-redirection.php

```diff
@@ -25,11 +25,13 @@ function jetpack_user_content_link_redirection() {
 <html>
 <head>
 <script>
+	var messageReceived = false;
```

Comment:
Nitpick, why not `let`? It'd avoid polluting  the `window` namespace. Or if you really like `var` you can put it inside an IIFE.

```js
(function () {
  var messageReceived = false;
  window.addEventListener( 'message', function(event) {
    ...
  });
})();

Reply:
I like `let` a lot, to be honest :D